app/views/comments/_new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with model: [current_project, Comment.new(commentable: commentable)], data: { behavior: 'add-comment' }, class: 'mt-3' do |f| %>
+<%= form_with model: [current_project, Comment.new(commentable: commentable)], data: { behavior: 'add-comment local-auto-save', auto_save_key: "comment-#{request.path}" }, class: 'mt-3' do |f| %>
```

Comment:
This looks like a rather long line, do you mind restructuring it for readability?